jdk/src/macosx/native/com/sun/media/sound/PLATFORM_API_MacOSX_Ports.cpp

```diff
@@ -22,6 +22,11 @@
  * or visit www.oracle.com if you need additional information or have any
  * questions.
  */
+/*
+ * ===========================================================================
+ * (C) Copyright IBM Corp. 2023 All Rights Reserved.
```

Comment:
Since the change is an OpenJDK backport, we shouldn't be adding an IBM copyright.

Reply:
I removed the IBM copyright.